src/core/onerror.js

```diff
@@ -0,0 +1,34 @@
+if ( defined.document ) {
```

Comment:
Can we wrap this module in a IEFE? Helps ensure `onErrorFnPrev` isn't visible anywhere else.